src/main/java/catan/domain/transfer/output/game/GameDetails.java

```diff
@@ -38,6 +39,7 @@ public GameDetails(GameBean game, int detailsRequesterId) {
         this.maxPlayers = game.getMaxPlayers();
         this.targetVictoryPoints = game.getTargetVictoryPoints();
         this.currentMove = game.getCurrentMove();
+        this.dice = game.getDiceFirstValue() == null ? null : new DiceDetails(game.getDiceValue(), game.getDiceFirstValue(), game.getDiceSecondValue());
```

Comment:
Andrii:
do we need to put dice details when game is not started yet? it is just a question for discussion, but we have take a decision and follow it for other JSON elements that are not populated yet.

Yurii:
as temporary decision, I refactored it so that "dice = null" in case if dices not thrown yet. IMHO, in future we shoud create separated JSONs for new/playing status and preparation/main stage

My questions to this commit:
1. Have we already discussed and taken the final decision? This solution can make troubles on front-end if it expects not-null value of JSON element 'dice' even if dice was not thrown.
2. Do you want to disucss format of response in different cases? we have already implemented the core and front-end architecture doesn't expect any significant changes in response, so if you would like to change the format we shuld discuss it as soon as possible but not later.